spec.html

```diff
@@ -39482,7 +39477,7 @@ <h1>AsyncFunctionStart ( _promiseCapability_, _asyncFunctionBody_ )</h1>
             1. Else,
               1. Assert: _result_.[[Type]] is ~throw~.
               1. Perform ! Call(_promiseCapability_.[[Reject]], *undefined*, &laquo; _result_.[[Value]] &raquo;).
-            1. [id="step-asyncfunctionstart-return-undefined"] Return.
+            1. [id="step-asyncfunctionstart-return-undefined"] Return NormalCompletion(~unused~).
           1. Push _asyncContext_ onto the execution context stack; _asyncContext_ is now the running execution context.
           1. Resume the suspended evaluation of _asyncContext_. Let _result_ be the value returned by the resumed computation.
```

Comment:
is this correct, given that 2 lines down, the returned value *is* used?

Reply:
I would not regard the assertion as using the value.

Reply:
additionally, the step ID says "return undefined" - is it not important that it's "undefined" versus an unused value?

Reply:
No, that's just an arbitrary label for making the later step reference work. I didn't change it because step IDs don't yet support `oldids` and I didn't want to break existing links.